build/networking/scripts/build_p4sde.sh

```diff
@@ -61,7 +61,7 @@ echo "Compiling p4-driver"
 git clone https://github.com/p4lang/p4-dpdk-target.git p4-driver
 pushd "$SDE/p4-driver"
 #Note: Below SHA needs to be updated when TDI code is open-sourced
-git checkout 780b3dfa205815e87f4580383cc37bfa30187f7c
+git checkout v22.07
```

Comment:
Build process needs to be updated with generic-flags for SDE to enable TDI

Reply:
P4 SDE team itself recommended not use those flags as that will limit it to only TDI. If you exclude those flags, it supports both bfrt and tdi.